### PR TITLE
DOC: Fix some minor mistakes in "LaTeX customization"

### PR DESCRIPTION
**Pull request by mgeier**
### Feature or Bugfix

- Documentation bugfix

### CODE REVIEW

**Comment by tk0miya** [comment]
@jfbu Could you review this please if you have time :-)

File: doc/latex.rst [hunk 1]
```diff
@@ -354,9 +354,9 @@ The available styling options
                                ``attentionBgColor``, ``dangerBgColor``,
                                ``errorBgColor``
 
-.. |warningborders| replace:: ``warningBorder``, ``cautionBorder``,
-                              ``attentionBorder``, ``dangerBorder``,
-                              ``errorBorder``
+.. |warningborders| replace:: ``warningborder``, ``cautionborder``,
+                              ``attentionborder``, ``dangerborder``,
+                              ``errorborder``
```
Comment (jfbu) [inline doc/latex.rst:359]: thanks @mgeier for catching this. I guess I intended to use the camelcase names in `sphinx.sty` but indeed (in `sphinxadmonition` environment definition, the `\dimexpr` line) it is done with lowercase names... too late to change that.

File: doc/latex.rst [hunk 2]
```diff
@@ -209,7 +209,7 @@ The available styling options
        the default changed from ``false`` to ``true``.
 
 ``verbatimcontinuedalign``, ``verbatimcontinuesalign``
-    default ``c``. Horizontal position relative to the framed contents:
+    default ``r``. Horizontal position relative to the framed contents:
```
Comment (jfbu) [inline doc/latex.rst:212]: yes, thanks for catching this!

**Review by jfbu — APPROVED** [review]
LGTM! thanks for catching these mistakes.

**Comment by jfbu** [comment]
Thanks I will backport to 1.8 branch.

**Pull request merged**